src/seed/requests.ts

```diff
@@ -0,0 +1,261 @@
+    const options: AxiosRequestConfig = {
+      method: 'post',
+      url,
+      headers: {
+        'Content-Type': 'application/json',
+        Date: (new Date()).toISOString()
+      },
+      data: {
+        ...request.body
+      }
+    }
+
+    return this.executeRequest(options)
+  }
+
+
+  public static async postParticipants(host: string, request: PostParticipantsRequest): Promise<AxiosResponse<any>> {
+    const url = `${host}/participants`
+    const options: AxiosRequestConfig = {
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      url,
+      data: {
+        ...request.body
+      }
+    }
+
+    return this.executeRequest(options)
+  }
+
+  public static async postParticipantsPositionAndLimits(host: string, request: PostParticipantsPositionAndLimitsRequest): Promise<AxiosResponse<any>> {
+    const url = `${host}/participants/${request.participantId}/initialPositionAndLimits`
+    const options: AxiosRequestConfig = {
+      method: 'post',
+      url,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      data: {
+        ...request.body
+      }
+    }
+
+    return this.executeRequest(options)
+  }
+
+  public static async postAccount(host: string, request: PostAccountRequest): Promise<AxiosResponse<any>> {
+    const url = `${host}/participants/${request.participantId}/accounts/${request.accountId}`
+    const options: AxiosRequestConfig = {
+      method: 'post',
+      url,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      data: {
+        ...request.body
+      }
+    }
+
+    return this.executeRequest(options)
+  }
+
+  public static async postEndpoint(host: string, request: PostEndpointsRequest): Promise<AxiosResponse<any>> {
+    const url = `${host}/participants/${request.participantId}/endpoints`
+    const options: AxiosRequestConfig = {
+      method: 'post',
+      url,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      data: {
+        ...request.body
+      }
+    }
+
+    return this.executeRequest(options)
+  }
+
+  public static async postSimulatorParty(host: string, request: PostSimulatorPartyRequest): Promise<AxiosResponse<any>> {
+    const url = `${host}/repository/parties`
+    const options: AxiosRequestConfig = {
+      method: 'post',
+      url,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      data: {
+        ...request.body
+      }
+    }
+
+    return this.executeRequest(options)
+  }
+
+  public static async postALSParticipant(host: string, request: PostALSParticipantRequest): Promise<AxiosResponse<any>> {
+    const url = `${host}/participants/${request.idType}/${request.idValueOrSubType}` + (request.idValue ? `/${request.idValue}` : '')
+    const options: AxiosRequestConfig = {
+      method: 'post',
+      url,
+      headers: {
+        'Accept': 'application/vnd.interoperability.participants+json;version=1',
```

Comment:
maybe we should take these headers (Accept, Content-Type) values from config, not hardcode here?